package.json

```diff
@@ -34,6 +34,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.4.5",
+    "@babel/helper-compilation-targets": "^7.10.4",
```

Comment:
Why is this needed?

Reply:
Not having that installed gave me this error: 🤷‍♂️ 
![image](https://user-images.githubusercontent.com/1352352/87834377-86278380-c88a-11ea-94e7-c9d162d08d34.png)

Reply:
Ah yeah you got a newer node version so it doesn't work, you need the newest babel version 😅 I think just upgrading the `@babel` deps to 7.10.4 should be sufficient